src/us/kbase/workspace/test/kbase/HandleAndShockTest.java

```diff
@@ -417,84 +427,62 @@ public void basicHandleTest() throws Exception {
 		 * object where they don't own the shock nodes, even with all other
 		 * privileges
 		 */
-		BasicShockClient bsc = new BasicShockClient(
-				new URL("http://localhost:" + SHOCK.getServerPort()), CLIENT1.getToken());
-		@SuppressWarnings("unused")
-		ShockACL acl = bsc.addToNodeAcl(new ShockNodeId(h1.getId()),
-				Arrays.asList(USER2), ShockACLType.ALL);
+		bsc.addToNodeAcl(new ShockNodeId(shock_id), Arrays.asList(USER2), ShockACLType.ALL);
 		String workspace2 = "basichandle2";
 		CLIENT2.createWorkspace(new CreateWorkspaceParams().withWorkspace(workspace2));
 		try {
 			CLIENT2.saveObjects(new SaveObjectsParams().withWorkspace(workspace2)
 					.withObjects(Arrays.asList(
 							new ObjectSaveData().withData(new UObject(handleobj)).withName("foo2")
 							.withType(HANDLE_TYPE))));
-			fail("saved object with bad handle");
-		} catch (ServerException e) {
-			assertThat("correct exception message", e.getMessage(),
-					is("An error occured while processing IDs: " +
-						"The Handle Service reported that at least one of " +
-						"the handles contained in the objects in this call " +
-						"is not accessible - it may not exist, or the " +
-						"supplied credentials may not own the node, or some " +
-						"other reason. The call cannot complete."));
+		} catch (ServerException se) {
+			System.out.println(se.getData());
+			throw se;
 		}
-		
-		bsc.removeFromNodeAcl(new ShockNodeId(h1.getId()),
+
+		bsc.removeFromNodeAcl(new ShockNodeId(shock_id),
 				Arrays.asList(USER2), ShockACLType.ALL);
-		
+
 		List<ShockUserId> oneuser = Arrays.asList(SHOCK_USER1);
 		List<ShockUserId> twouser = Arrays.asList(SHOCK_USER1, SHOCK_USER2);
-		
-		ShockNode node = bsc.getNode(new ShockNodeId(h1.getId()));
-		
+
 		checkReadAcl(node, oneuser);
 
 		// test that user2 can get shock nodes even though permissions have
 		// been removed when user2 can read the workspace object
 		CLIENT1.setPermissions(new SetPermissionsParams().withWorkspace(workspace)
 				.withUsers(Arrays.asList(USER2)).withNewPermission("r"));
 		//get objects2
-				
 		ObjectData ret1 = CLIENT2.getObjects2(new GetObjects2Params()
 				.withObjects(Arrays.asList(new ObjectSpecification()
 					.withWorkspace(workspace)
 					.withObjid(1L)))).getData().get(0);
 		checkExternalIDError(ret1.getHandleError(), ret1.getHandleStacktrace());
-		
-		checkReadAcl(node, twouser);
```

Comment:
Why are these lines removed?

Reply:
And all the other similar lines below that are removed.

Reply:
👍